--- jquery-uploadify-3.2.1/jquery.uploadify.js~	2013-08-06 13:02:03.578771849 +0300
+++ jquery-uploadify-3.2.1/jquery.uploadify.js	2013-08-06 13:06:41.567624860 +0300
@@ -584,7 +584,7 @@
 			if (settings.itemTemplate == false) {
 				settings.itemTemplate = '<div id="${fileID}" class="uploadify-queue-item">\
 					<div class="cancel">\
-						<a href="javascript:$(\'#${instanceID}\').uploadify(\'cancel\', \'${fileID}\')">X</a>\
+						<a href="javascript:jQuery(\'#${instanceID}\').uploadify(\'cancel\', \'${fileID}\')">X</a>\
 					</div>\
 					<span class="fileName">${fileName} (${fileSize})</span><span class="data"></span>\
 					<div class="uploadify-progress">\
@@ -910,4 +910,4 @@
 
 	}
 
-})($);
\ No newline at end of file
+})(jQuery);
